subsys/bluetooth/shell/bt.c

```diff
@@ -1217,6 +1219,261 @@ static int cmd_adv_info(const struct shell *shell, size_t argc, char *argv[])
 
 	return 0;
 }
+
+#if defined(CONFIG_BT_PER_ADV)
+static int cmd_per_adv(const struct shell *shell, size_t argc, char *argv[])
+{
+	struct bt_le_ext_adv *adv = adv_sets[selected_adv];
+
+	if (!adv) {
+		shell_error(shell, "No extended advertisement set selected");
+		return -EINVAL;
+	}
+
+	if (!strcmp(argv[1], "off")) {
+		if (bt_le_per_adv_enable(adv, false) < 0) {
+			shell_error(shell,
+				    "Failed to stop periodic advertising");
+		} else {
+			shell_print(shell, "Periodic advertising stopped");
+		}
+	} else if (!strcmp(argv[1], "on")) {
+		if (bt_le_per_adv_enable(adv, true) < 0) {
+			shell_error(shell,
+				    "Failed to start periodic advertising");
+		} else {
+			shell_print(shell, "Periodic advertising started");
+		}
+	} else {
+		shell_error(shell, "Invalid argument: %s", argv[1]);
+		return -EINVAL;
+	}
+
+	return 0;
+}
+
+static int cmd_per_adv_param(const struct shell *shell, size_t argc,
+			     char *argv[])
+{
+	struct bt_le_ext_adv *adv = adv_sets[selected_adv];
+	struct bt_le_per_adv_param param;
+	int err;
+
+	if (!adv) {
+		shell_error(shell, "No extended advertisement set selected");
+		return -EINVAL;
+	}
+
+	if (argc > 1) {
+		param.interval_min = strtol(argv[1], NULL, 10);
```

Comment:
base 16 for interval. (since other interval arguments use it).

Reply:
Nice catch!